src/common/utils/is-node-environment/index.ts

```diff
@@ -0,0 +1,11 @@
+export const isNodeEnvironment = () => {
+  if (typeof window === 'undefined') {
+    return true;
+  }
+
+  if (window.name === 'nodejs') {
```

Comment:
JSDOM doesn't set `window.name` to `nodejs` as of version 12 😕

Reply:
@albertogasparin this will be a problem in RLL as well then, so we need a fix, this issue https://github.com/jsdom/jsdom/issues/1537 states that we should use 

```
navigator.userAgent.includes("Node.js") || navigator.userAgent.includes("jsdom")
```

But I think this is so jsdom specific... it depends on whether we are comfortable saying that on the server jsdom is a hard dependency.

Reply:
We could potentially use https://github.com/flexdinesh/browser-or-node